webextension/popup.html

```diff
@@ -8,21 +8,30 @@
 <body>
 
   <div class="panel onboarding onboarding-panel-1 hide" id="onboarding-panel-1">
-    <img alt="Container Tabs Overview" src="/img/onboarding-1.png" />
-    <h3>A better way to manage all the things you do online</h3>
-    <p>Use containers to organize tasks, manage accounts, and keep your focus where you want it.</p>
-    <p class="spacing" ><a href="#">Learn more</a></p>
-    <a href="#" id="onboarding-next-button" class="button expanded">Next</a>
+    <img class="onboarding-img" alt="Container Tabs Overview" src="/img/onboarding-1.png" />
+    <h3 class="onboarding-title">A better way to manage all the things you do online</h3>
+    <p>
+      Use containers to organize tasks, manage accounts, and keep your focus where you want it.
+      <br/>
+      <a href="#">Learn more</a>
+    </p>
+    <a href="#" id="onboarding-start-button" class="button expanded onboarding-button">Get Started</a>
   </div>
 
 
   <div class="panel onboarding onboarding-panel-2 hide" id="onboarding-panel-2">
     <img alt="How Containers Work" src="/img/onboarding-2.png" />
     <h3>Put containers to work for you.</h3>
     <p>Features like color-coding and separate container tabs help you find things easily, focus your attention, and minimize distractions.</p>
-    <a href="#" id="onboarding-done-button" class="button expanded spacing">Done</a>
+    <a href="#" id="onboarding-next-button" class="button expanded spacing">Next</a>
   </div>
 
+  <div class="panel onboarding onboarding-panel-3 hide" id="onboarding-panel-3">
+    <img alt="How Containers Work" src="/img/onboarding-3.png" />
+    <h3>A place for everything, and everything in its place.</h3>
```

Comment:
Do we need to add `class="onboarding-img"` and `class="onboarding-title"` here as well (similar to panel-1 above)?